cetmix_tower_server/views/cx_tower_command_view.xml

```diff
@@ -67,6 +67,7 @@
                                 widget="many2many_tags"
                                 options="{'color_field': 'color'}"
                             />
+                            <field name="secret_ids" widget="many2many_tags" />
```

Comment:
Please make this field visible only to the members of the "Manager" and "Root" groups.